src/client/l10n/currency.ts

```diff
@@ -0,0 +1,23 @@
+import { MarketLabel } from './locales'
+
+export type CurrencyData = {
+  currencySymbol: string
+  currencyCode: string
+}
+
+type Currency = Record<MarketLabel, CurrencyData>
+
+export const currency: Currency = {
+  SE: {
+    currencySymbol: 'kr',
+    currencyCode: 'SEK',
+  },
+  NO: {
+    currencySymbol: 'kr',
+    currencyCode: 'NOK',
+  },
+  DK: {
+    currencySymbol: 'kr.',
+    currencyCode: 'DKK',
+  },
+}
```

Comment:
On displaying currency properly: I had looked into this before and there is a good browser API that achieves this so we don't have to make it ourselves. The browser API also handles the position of the currency in the price string (eg `3kr` vs `$3`).

https://hedvig.atlassian.net/browse/GRW-559

Reply:
Ah cool, I hadn't seen that Jira issue. But also saw that @robinandeer commented about having the logic in Giraffe. Did you discuss this further? 🤔 Also, did you do any further investigations into fallback solutions and whether or not we need one of those? Sounds super neat if it would just work with that browser API 👌

Reply:
Hmm, yeah I think the giraffe-approach is too far out to consider now

The browser API usually only supports English locale 🙄  but we could investigate if adding other locales is worth it. But yeah, I would do that as a separate PR/project.

Reply:
But doesn't it support all (or most locales)? Or seems like it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) at least? Makes a lot of sense to me to use this, since as @mehdielket pointed out that also solves position of currency symbol, plus if there should be a comma or dot or whitespace as delimiter. Or am I just missing something from skimming the MDN page?

Reply:
Hmm maybe it does -- that would be amazing :)

Reply:
OK so I've made a try to use the JS Internationalization API for this and it seems to work. Only annoying thing is that the `Intl.NumberFormat()` constructor of course uses a format for the locale arguments than we don't use anywhere else. I solved this in the most "dumb" but easy way and added a `currencyLocale` to the `locales` objects. Felt that all "smart" solutions to use other locale properties that we already have would most likely be shooting ourselves in the foot.